docker/gaffer-integration-tests/Dockerfile

```diff
@@ -1,14 +1,30 @@
+# Copyright 2021-2023 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 ARG BUILDER_IMAGE_NAME=maven
 ARG BUILDER_IMAGE_TAG=3.8.4-jdk-8
 
 FROM ${BUILDER_IMAGE_NAME}:${BUILDER_IMAGE_TAG}
 ARG GAFFER_GIT_REPO=https://github.com/gchq/Gaffer.git
 ARG GAFFER_VERSION=develop
+ARG ACCUMULO_VERSION=1.9.3
+ENV ACCUMULO_VERSION=$ACCUMULO_VERSION
```

Comment:
Is the default for if this was run without an `.env` file? Would it actually work in that situation or would other things be missing?